Examples/UIExplorer/UIExplorerApp.android.js

```diff
@@ -39,11 +39,52 @@ const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
 
 var DRAWER_WIDTH_LEFT = 56;
 
+function PathActionMap(path: string): ?Object {
+  // Warning! Hacky parsing for example code. Use a library for this!
+  const exampleParts = path.split('/example/');
+  const exampleKey = exampleParts[1];
+  if (exampleKey) {
+    if (!UIExplorerList.Modules[exampleKey]) {
+      Alert.alert(`${exampleKey} example could not be found!`);
+      return null;
+    }
+    return UIExplorerActions.ExampleAction(exampleKey);
+  }
+  return null;
+}
+
+function URIActionMap(uri: ?string): ?Object {
+  // Warning! Hacky parsing for example code. Use a library for this!
+  if (!uri) {
+    return null;
+  }
+  const parts = uri.split('rnuiexplorer:/');
+  if (!parts[1]) {
+    return null;
+  }
+  const path = parts[1];
+  return PathActionMap(path);
+}
+
+type Props = {
+  openExampleName: String,
+};
+
 class UIExplorerApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this._handleOpenExample = this._handleOpenExample.bind(this);
+    this.state = {openExampleName: props.openExampleName};
+  }
+
   componentWillMount() {
     BackAndroid.addEventListener('hardwareBackPress', this._handleBackButtonPress.bind(this));
   }
 
+  componentDidMount() {
+    setTimeout(this._handleOpenExample, 500);
```

Comment:
There were two approaches tried. For iOS I could make use of deep linking by triggering this call:

```
[RCTLinkingManager application:application openURL:url
                      sourceApplication:sourceApplication annotation:annotation];
```

While passing in the expected link, ex: rnuiexplorer://example/ImageExample

Android was a different story. I couldn't get basic deep links to work within Android. Android deep linking has an existing race condition that @ericvicenti is aware of. So I took a slightly different approach componentDidMount + directly call an action on NavigationExperimental. Since I was doing this for Android, I decided to do the same on iOS. However, it's a hack.

I'll put this note in the code as well and rename.